index.js

```diff
@@ -201,6 +201,7 @@ var transfer = (
   ts,
   channelid
 ) => {
+  if (user != 'UJYDFQ2QL') return;
```

Comment:
the fuck is this shit

Reply:
its to test bro, so only i can make a transaction, and we don't bring banker fully online until I verify that it works